google/cloud/aiplatform/models.py

```diff
@@ -69,6 +72,8 @@
 _DEFAULT_MACHINE_TYPE = "n1-standard-2"
 _DEPLOYING_MODEL_TRAFFIC_SPLIT_KEY = "0"
 _SUCCESSFUL_HTTP_RESPONSE = 300
+_RAW_PREDICT_MODEL_ID_KEY = "X-Vertex-AI-Deployed-Model-Id"
```

Comment:
Probably call this ”deployed model id” instead of “model id” (which sometimes refers to the last part of the resource full name).

Reply:
updated variable name